databuilder/setup.py

```diff
@@ -85,14 +78,14 @@
     'mysqlclient>=1.3.6,<3'
 ]
 
-all_deps = requirements + kafka + cassandra + glue + snowflake + athena + \
+all_deps = requirements + requirements_dev + kafka + cassandra + glue + snowflake + athena + \
```

Comment:
also why not put common into databuilder given eventually we should align model definition between common and databuilder?

Reply:
requirements_common contain mostly shared dependencies for Flask services (+ obviously amundsen-common). during rfc we decided to exclude databuilder from sharing those (which would make sense if we decide not to pin any deps in databuilder, while in requirements_common flask and other deps are need to be pinned).